templates/header.hbs

```diff
@@ -0,0 +1,15 @@
+    <header>
```

Comment:
add the classes: class="header component"

Reply:
I have used class='header'